BE/app/services/competitor_channel_service.py

```
@@ -708,3 +709,402 @@ async def analyze_recent_video(
+                    else:
+                        # 전체 분석 수행 (자막 fetch + 전체 LLM)
+                        await CompetitorChannelService.analyze_recent_video(
+                            db, video.video_id, user_id
+                        )
+                        analyzed_count += 1
+
+                except Exception as e:
+                    logger.warning(f"자동 분석 실패 ({video.video_id}): {e}")
+                    continue
+
+        total = analyzed_count + reused_count
```

Ruff (0.14.14)

[error] Local variable `total` is assigned to but never used

Remove assignment to unused variable `total`

(F841)